sdk/tests/js/webgl-test-utils.js

```diff
@@ -2619,6 +2619,28 @@ var getSupportedExtensionWithKnownPrefixes = function(gl, name) {
   }
 };
 
+/**
+ * @param {WebGLRenderingContext} gl The WebGLRenderingContext to use.
+ * @param {string} name Name of extension to look for.
+ * @param {boolean} extensionEnabled True if the extension was enabled successfully via gl.getExtension().
+ */
+var runExtensionSupportedTest = function(gl, name, extensionEnabled) {
```

Comment:
I would sooort of rather we do this test for all extensions in a single test file, rather than spreading this boilerplate out across the extension test files, but centralizing it here at least helps.

Reply:
Agreed, that would help.

Additionally, in Chromium there have been regressions where certain extensions suddenly stopped being supported. In order to catch those, we had to add tests outside this harness which assert that certain extensions must be supported on certain platforms.

Reply:
FWIW our tests for this are here:
https://dxr.mozilla.org/mozilla-central/source/dom/canvas/test/webgl-mochitest/ensure-exts